code/game/machinery/kitchen/microwave.dm

```diff
@@ -235,7 +235,7 @@
 *   Microwave Menu Handling/Cooking
 ************************************/
 
-/obj/machinery/microwave/proc/cook()
+/obj/machinery/microwave/proc/cook(var/mob/living/user)

```

Comment:
Why is this needed?

Reply:
Ignore, was testing around with a function and commited it. However after talking with skull, he apparently wants to stick with butchering the animal first as microwave code sucks ass in general.